components/Dataservices/DescribeDataservice.vue

```diff
@@ -336,10 +336,19 @@
           <LinkedToAccordion
             class="fr-fieldset__element"
             :accordion="contactPointAccordionId"
-            @blur="touch('contact_point')"
+            @blur="touch('contact_points')"
           >
+            <template v-for="(contact_point, index) in form.contact_points">
+              <ContactPointSelect
+                v-if="contact_point.role == 'contact'"
+                :key="'id' in contact_point ? contact_point.id : index"
+                v-model="form.contact_points[index]"
+                :organization="form.owned?.organization"
+              />
+            </template>
             <ContactPointSelect
-              v-model="form.contact_point"
+              v-if="form.contact_points.length === 0"
```

Comment:
I think this condition does not work? When adding a contact point a new contact

Reply:
This condition is to handle the case when there is no contact point yet.

When there is one, the template above contains the contact point